File: Prak503/PRAK503-2210817210021-TRISNACAHYAPERMADI.py
def maksimal (x, y) :
    if x > y :
        return x
    else :
        return y
def minimal (x, y) :
    if x < y :
        return x
    else :
        return y
batas = 0; maks = -100000; minim = 100000
bilangan = int (input())
while batas < bilangan :
    nilai = map(int, input().split())
    for item in nilai :
        maks = maksimal(maks, item)
        minim = minimal(minim, item)
        batas += 1
print (maks, minim)
    

File: Prak503/PRAK503-2210817210021-TRISNACAHYAPERMADI.c
#include <stdio.h>
int maksimal(int a, int b){
    if (a > b){
    return a;
    }
    else{
        return b;
    }
}
int minimal (int a, int b){
    if (b < a){
    return b;
    }
    else{
        return a;
    }
}
int main (void)
{
    int batas = 0;
    int maks = -100000;
    int minim = 100000;
    int bilangan;
    scanf ("%i", &bilangan);
    while (batas < bilangan){
        int nilai;
        scanf ("%i", &nilai);
    maks = maksimal(maks, nilai);
    minim = minimal(minim, nilai);
    batas++;
    }
    printf ("%i %i", maks, minim);
}